### PR TITLE
bnpl orders

### PR DESCRIPTION
**Pull request by darmhoo**
Add bnpl filter to new_order api

### CODE REVIEW

File: app/Http/Filters/NewOrderFilter.php [hunk 1]
```diff
@@ -275,4 +276,9 @@ public function financed_by(string $financed_by)
     {
         $this->builder->where('financed_by', $financed_by);
     }
+    public function bnplOrders(bool $isBNPL = true ){
```
Comment (teebarg) [inline app/Http/Filters/NewOrderFilter.php:279]: Can we have a scenario where $isBNPL could be false?

Comment (darmhoo) [reply app/Http/Filters/NewOrderFilter.php:279]: No. i should remove the condition

File: app/Http/Filters/NewOrderFilter.php [hunk 2]
```diff
@@ -268,11 +269,16 @@ public function businessTypeGroup(string $group)
...
 
-      /**
+    /**
      * @param int $id
      */
     public function financed_by(string $financed_by)
     {
         $this->builder->where('financed_by', $financed_by);
     }
+    public function bnplOrders(bool $isBNPL = true)
```
Comment (teebarg) [inline app/Http/Filters/NewOrderFilter.php:279]: Can we remove the argument then?

Comment (darmhoo) [reply app/Http/Filters/NewOrderFilter.php:279]: done

**Review by teebarg — APPROVED** [review]

**Pull request merged**